### DISCUSSION TITLE
the Cardinal point is not displayed In Japanese in version 0.21.1

### DISCUSSION THREAD

**Discussion by AstroFuyuko** [Stellarium/stellarium#2517] [Q&A] [answered]
Hi everyone.
I am Japanese.
I am using Ver **0.21.1** Stellarium.

Please tell me about cardinal point using Japanese.

I changed below. 
[ Configuration menu ] - [main panel ] - [ Program  Language ]  ---> Japanese   ... 日本語(にほんご)

But cardinal point didn't change in **Ver 0.21.1**
That could change in version **0.19.3** , correct cardinal point  ("東","西","南","北") was displayed in Japanese.

Does anyone know how to solve this problem?

Is this Bug?
If it is Bug , I hope Developers can handle the problem.

Thanks very much
Fuyuko

**Comment by gzotti** [comment, marked as answer]
We cannot comment on old versions. 0.22.2 looks like this, and it seems OK to me:
![stellarium-068](https://user-images.githubusercontent.com/18099873/179393455-c31f3863-680a-480f-952d-d2a2aaa45aa2.jpg)

**Reply by AstroFuyuko** [reply]
MR./Ms. [gzotti](https://github.com/gzotti) 

Thank you for your replaying.
I installed  0.22.2 .
the Cardinal point in Japanese is displayed!!!

But a new problem has occurred.
Any Scripts  that was been working in  Ver 0.21.1 doesn't work in 0.22.2 .
I will check which command doesn't work in 0.22.2 .

Anyway The problem with this thread has been resolved.
Thank you very much.
